guesser.rb

```diff
@@ -0,0 +1,35 @@
+answer_again = "yes"
+
+while answer_again == "yes" do 
+    puts "can you think of a number between 1 and 100?"
+    lower_bound = 1
+    upper_bound = 100
+    guess = rand (lower_bound..upper_bound)
```

Comment:
Using `rand` again could work, but the directions want you to focus a bit more on efficiency for this one per the directions:
> Make the program efficient -- basically a binary search.

For good practice, the best way to do that is get the midpoint by taking the average of the lowest and highest possible answers.

Reply:
Update: I think they want more of something like this (with the average):

`guess = (lower_bound + upper_bound) / 2`